package.json

```diff
@@ -19,8 +22,9 @@
     "docs": "docsify serve",
     "browser-test": "airtap --local 8080 test/browser*test.js",
     "lint": "eslint .",
-    "test": "npm run lint && tap --100 test/*test.js test/*/*test.js",
-    "test-ci": "npm run lint && tap test/*test.js test/*/*test.js --coverage-report=lcovonly",
+    "test": "npm run lint && tap --100 test/*test.js test/*/*test.js && npm run test-types",
+    "test-ci": "npm run lint && tap test/*test.js test/*/*test.js --coverage-report=lcovonly && npm run test-types",
```

Comment:
@davidmarkclements Please let me know if you want this removed.

Reply:
yeah can you please remove test-ci and make it work as before with your additions

Reply:
@davidmarkclements Do you mean removing entire "test-ci", or just "npm run test-types" out of it?